### PR TITLE
Mcho/add command ungraftmoduleonhook

### PR DESCRIPTION
**Pull request by Mch0**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ php bin/console pr:mo install fop_console
 * `fop:generate:htaccess` Generate the .htaccess file
 * `fop:generate:robots`   Generate the robots.txt file
 * `fop:add-hook` : Create a new hook in database
+* `fop:graft-module-on-hook` : Graft module on specific hook
```
Comment (nenes25) [inline README.md:36]: Still the old wording ?

**Review by nenes25 — CHANGES_REQUESTED** [review]
Just update the wording :) 
For the code everything seems good.

File: src/Commands/UnhookModule.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+class UnhookModule extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setName('fop:unhook-module')
+            ->setDescription('Detach module from hook')
+            ->setHelp('This command allows you to dettach a module from one hook');
```
Comment (okom3pom) [inline src/Commands/UnhookModule.php:35]: Only one T

**Review by okom3pom — CHANGES_REQUESTED** [review]

**Review by okom3pom — APPROVED** [review]

**Review by nenes25 — APPROVED** [review]

**Review by SebSept — APPROVED** [review]
Super, Merci @Mch0 :)

**Pull request merged**